### PR TITLE
Update use-dmarc-to-validate-email.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another line in the section to create transport rule for all users reject emails that fail org dmarc policy.

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9471

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md [hunk 1]
```diff
@@ -233,6 +233,8 @@ Microsoft 365 is configured like this because some legitimate email may fail DMA
 
 - Admins create an Exchange mail flow rule (also known as a transport rule) for all users that allows messages for those particular senders.
 
+- Admins create an Exchange mail flow rule (also known as a transport rule) for all users reject emails that fail your organizations DMARC Policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dmarc-to-validate-email.md]:
```suggestion
- Admins create an Exchange mail flow rule (also known as a transport rule) for all users for rejected emails that fail your organization's DMARC policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by chrisda** [comment]
Too old. Aticle renamed. I had copied the one-line change into a new PR.

Closing

**Pull request closed without merging**